src/app/app.component.ts

```diff
@@ -217,33 +213,15 @@ export class MyApp implements OnInit, OnDestroy {
     });
   }
 
-  // Verify if any account registered is a delegate
```

Comment:
I don't see a good reason to remove this, the purpose is to save mobile data

Reply:
hmm I think I don't understand you? I removed this, because the only thing which it did was set `isDelegate` to true on the wallets. However the `isDelegate` state is now set when creating / importing a wallet and also if the user has become a delegate (refresh method). So this method shouldn't be required anymore?

Reply:
Well, if you register a delegate in the desktop wallet will only be displayed when you open the wallet

Reply:
Well this only fetches the 102 delegates anyway I think. But you're still right and it probably doesn't hurt to have it. I will readd it again.

Reply:
You're right, this change to only 102 delegates was made by a PR after I deployed. So this method may not make more sense 😥

Reply:
I see. Should i do anything there or leave it like it is?
Another alternative would be to check the delegate status for all wallets in the WalletsListPage (if they are not already a delegate) in the `OnInit` method. However this would mean potentially unneeded requests. What do you thik?

Reply:
Nah, your solution is good, just add the refresh condition

Reply:
Sorry: Which refresh condition are you talking about? 😅

Reply:
https://github.com/ArkEcosystem/ark-mobile/pull/77#discussion-diff-169168441L388

Reply:
=> that means I did not change anything here (see 4ffa368)